server/controllers/anoynmous.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const router = express.Router();
+
+const Anonymous = require("../models/anonymous");
```

Comment:
the file in models/ has a typo so nor sure if the require is working here for the model